aiida/backends/general/abstractqueries.py

```diff
@@ -7,16 +7,11 @@
 # For further information on the license, see the LICENSE.txt file        #
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
-from __future__ import division
-from __future__ import print_function
-from __future__ import absolute_import
 
 from abc import ABCMeta
-import six
 
 
-@six.add_metaclass(ABCMeta)
-class AbstractQueryManager(object):
+class AbstractQueryManager:
```

Comment:
Should the metaclass actually be removed, or just put with new syntax? Could also inherit from ``ABC`` instead.